requirements-dev.txt

```diff
@@ -0,0 +1 @@
+gunicorn==20.0.4
```

Comment:
I don't think this should be in requirements-dev.txt. The package gunicorn is needed when running. I'm not sure what we buy by separating this from requirements.txt as both have to be installed in the Dockerfile and therefore are in the docker image.